src/doc.js

```diff
@@ -150,7 +150,7 @@ var prototype = node.derive({
                         baseline: line.baseline,
                         left,
                         content: {
-                            text: text.run.text.trim() + '&#160;'.repeat( word.space.length ),
+                            text: text.run.text.trim().replace( /&(?!#?[a-z0-9]+;)/, '&amp;' ) + '&#160;'.repeat( word.space.length ),
```

Comment:
Can you please add a comment explaining what this does (I had to use https://regex101.com/r/4vhX9N/1 to understand what this does). If this code is from StackOverflow, please add a link to the answer.